--- a/rsax931.py	2019-01-17 13:14:35.579005000 +0000
+++ b/rsax931.py	2019-01-17 13:14:40.242308014 +0000
@@ -46,7 +46,7 @@
             lib = glob.glob('/opt/local/lib/libcrypto.so*') + glob.glob('/opt/tools/lib/libcrypto.so*')
             lib = lib[0] if len(lib) > 0 else None
         if lib:
-            return cdll.LoadLibrary(lib)
+            return cdll.LoadLibrary("libcrypto.so")
         raise OSError('Cannot locate OpenSSL libcrypto')
	 